StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedViewRepresentable.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  EmbeddedViewRepresentable.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 1/30/25.
+//
+
+import SwiftUI
+@_spi(STP) import StripeCore
+
+struct EmbeddedViewRepresentable: UIViewRepresentable {
+    @ObservedObject var viewModel: EmbeddedPaymentElementViewModel
+
+    init(viewModel: EmbeddedPaymentElementViewModel) {
+        self.viewModel = viewModel
+        STPAnalyticsClient.sharedClient.addClass(toProductUsageIfNecessary: EmbeddedSwiftUIProduct.self)
+    }
+    
+    public func makeUIView(context: Context) -> UIView {
+        let containerView = UIView()
+        containerView.backgroundColor = .clear
+        containerView.layoutMargins = .zero
+
+        guard let embeddedPaymentElement = viewModel.embeddedPaymentElement else { return containerView }
```

Comment:
Should this error?

Reply:
Yeah, that's a good thought. I think it should! This case should only happen if you show the EmbeddedPaymentElementView in your SwiftUI view before isLoaded is true on the view model. According to the [docs](https://developer.apple.com/documentation/swiftui/uiviewrepresentable/makeuiview(context:)) makeUIView is only ever called once, so this looks safe and appears so from testing in an example integration.